examples/examples.yaml

```diff
@@ -1,6 +1,7 @@
 - path: "models/file"
   slow: ["choropleth.py"]
   no_js: ["maps_cities.py", "maps.py", "trail.py"]
+  no_diff: ["graphs.py"]
```

Comment:
You're right that doing no_diff reduces its test value. I'll modify the example to use the two deterministic layout algos and remove the no_diff flag.